spec.html

```diff
@@ -872,10 +872,10 @@ <h1>Mathematical Operations</h1>
       </ul>
 
       <p>In the language of this specification, numerical values and operations (including addition, subtraction, negation, multiplication, division, and comparison) are distinguished among different numeric kinds using subscripts. The subscript <sub><dfn id="𝔽">𝔽</dfn></sub> refers to Numbers, and the subscript <sub><dfn id="ℝ">ℝ</dfn></sub> refers to mathematical values. A subscript is used following each numeric value and operation.</p>
-      <p>For brevity, the <sub>𝔽</sub> subscript can be omitted on Number values--a numeric value with no subscript is interpreted to be a Number. An operation with no subscript is interpreted to be a Number operation, unless one of the parameters has a particular subscript, in which case the operation adopts that subscript. For example, 1<sub>ℝ</sub> + 2<sub>ℝ</sub> = 3<sub>ℝ</sub> is a statement about mathematical values, and 1 + 2 = 3 is a statement about Numbers.</p>
```

Comment:
i have a light preference to include the spaces around em dashes; it makes the raw ecmarkup more readable,

Reply:
[Most style guides](https://www.quickanddirtytips.com/education/grammar/em-dash-space) do not include spaces around em-dashes, and the spec currently does not do so either (despite @jmdyck's comment).

Reply:
Sure, but many style guides also recommend omitting the Oxford comma, or avoiding singular “they” - we can certainly make our own choice here. Either way, for this PR, we should follow whatever pattern is most used for em dashes - which below comments imply is “with spaces”.